components/Input/react/ReadMe.md

```diff
@@ -0,0 +1,94 @@
+# Input
+
+## Install
+
+```bash
+npm install @cypress-design/react-input
+```
+
+or with yarn
+
+```bash
+yarn add @cypress-design/react-input
+```
+
+## Usage
+
+```ts
+import Input from '@cypress-design/react-input'
+```
+
+```jsx live
+import {
+  default as Input,
+  VariantClassesTable,
+  SizeClassesTable,
+} from '@cypress-design/react-input'
+
+const Contrast = ({ children }) => (
+  <div className="bg-white p-[2px] rounded">{children}</div>
+)
```

Comment:
Since we had to create this blob, my guts tell me that "by default" inputs should have a background. That's an opinion though.

Why did you decide to not add it?